src/main/java/fi/metatavu/dcfb/server/rest/LocationsApiImpl.java

```diff
@@ -192,6 +202,12 @@ public Response updateLocation(UUID locationId, Location payload) throws Excepti
 
   @Override
   public Response deleteLocation(UUID locationId) throws Exception {
+    if (!isRealmUser()) {
+      return createForbidden("Anonymous users can not delete locations");
+    }
+
+    // TODO: Add resource based permission check
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=13&issues=AWQwvtHSbXXTsx7OQ1wS&open=AWQwvtHSbXXTsx7OQ1wS)

[ISSUE_KEY]: AWQwvtHSbXXTsx7OQ1wS